src/deploy/osp_deployer/deployer.py

```diff
@@ -262,6 +276,45 @@ def deploy():
     logger.info("log : /auto_results/ ")
     sys.exit(ret_code)
 
+def validate_performance_optimization_params(
```

Comment:
Could you move that function into settings_sanity.py,   an add a call to  it in checkpoints.py :: verify_deployer_settings

Reply:
Validations moved to profile.py
Fixed